db/sql/verify/buildings_lds/functions/populate_buildings_lds.sql

```diff
@@ -25,15 +25,13 @@ END $$;
 
 DO $$
 BEGIN
-
     PERFORM proname, proargnames, prosrc 
     FROM pg_proc
     WHERE proname = 'nz_building_outlines_all_sources_insert'
-    AND prosrc LIKE '%suburb_3rd%';
-    IF NOT FOUND THEN
-        RAISE EXCEPTION 'suburb_3rd not found.';
+    AND prosrc LIKE '%deleted_in_production.building_outline_id IS NULL%';
+    IF FOUND THEN
+        RAISE EXCEPTION 'Building outlines that are deleted in production shouldn't be excluded.';
```

Comment:
You've got a syntax error here with the ', just putting the whole string in "" should fix this.

Also not sure if it is something you've discussed with Daniel but should we be keeping the older code and adding this or do we not need this code anymore?